source/_integrations/sws12500.markdown

```diff
@@ -0,0 +1,57 @@
+---
+title: "Sencor SWS 12500 Weather Station"
+description: "Integration of SWS 12500 weather station."
+ha_release: "2023.1.8"
+ha_category: Weather
+ha_iot_class: "Local Push"
+ha_config_flow: "true"
+ha_codeowners:
+  - '@schizza'
+ha_domain: sws12500
+ha_platforms:
+  - sensor
+---
+*Integrates your weather station seamlessly into Home Assistant.*
+
+This integration will listen for data from your station and passes them into respective sensors.
+Also you are able to push data to Windy API.
+
+## Requirements
+
+- [Sencor SWS 12500 Weather Station](https://www.sencor.cz/profesionalni-meteorologicka-stanice/sws-12500)
+- Station set to send data directly to Home Assistant
```

Comment:
As a new user, I would be confused by this. Line, what does it mean? What do I have to do?